browser/ui/webui/brave_webui_source.cc

```diff
@@ -264,6 +264,8 @@ void CustomizeWebUIHTMLSource(const std::string &name, content::WebUIDataSource*
         { "allowTip", IDS_BRAVE_UI_ALLOW_TIP },
         { "amount", IDS_BRAVE_UI_AMOUNT },
         { "backup", IDS_BRAVE_UI_BACKUP },
+        { "backupNow", IDS_BRAVE_UI_BACKUP_NOW },
```

Comment:
why is this needed as we only display notification in extensions?

Reply:
removed